src/api/order.js

```diff
@@ -78,6 +79,7 @@ export function getOrder(orderID : string, { facilitatorAccessToken, buyerAccess
         return callRestAPI({
             accessToken: facilitatorAccessToken,
             url:         `${ ORDERS_API_URL }/${ orderID }`,
+            eventName:   `v2_checkout_orders_get`,
```

Comment:
I noticed we are doing this else where in this file, but I think we should stick with single quotes and only use string interpolation when we have variables in the string.
```suggestion
            eventName:   'v2_checkout_orders_get',
```